circe-akka-serializer/src/test/resources/application.conf

```diff
@@ -9,9 +9,23 @@ akka.actor {
   allow-java-serialization = off
 }
 
-#Uncomment below to enable logging
-
+# Uncomment chosen settings below to enable debug logging OR change compression settings
 #akka.loglevel = "DEBUG"
-#org.virtuslab.ash {
-#  verbose-debug-logging = on
-#}
+org.virtuslab.ash {
+  circe {
+    # enables debug logging
+    #verbose-debug-logging = on
+
+    # settings for compression of the payload
+    compression {
+      # Compression algorithm.
+      # - off  : no compression
+      # - gzip : using common java gzip
+      algorithm = off
```

Comment:
One more thing! as we discussed on Slack: when `algorithm = off`, will the deserializer still be able to **read** compressed data? 🤔 this is a hard requirement to cater for rollouts, when for a short time old nodes without compression support might work alongside new nodes with compression support.

Reply:
Pls document in reference.conf

Reply:
yes, it will be able to read either data compressed with Gzip or uncompressed. I will add description in the reference.conf